pan/legacy.pan

```diff
@@ -1,3 +1,12 @@
 declaration template pan/legacy;
 
 type legacy_binary_affirmation_string = string with match(SELF, "^(yes|no)$");
+
+function is_yes_no_true_false = {
+    v = ARGV[0];
+    if (is_boolean(v)) return(true);
+    if (is_string(v) && match(v, "^(yes|no)$")) return(true);
```

Comment:
I'd suggest not to add a deprecated message until we are ready to do the effort to do the migration (and first until we evaluated how much work is involved). This would pollute a lot the compilation messages.